Misc/NEWS.d/next/Core and Builtins/2024-06-04-08-26-25.gh-issue-120010._z-AWz.rst

```diff
@@ -0,0 +1,2 @@
+Correct invalid corner cases (resulted in ``(nan+nanj)`` output) in complex
+multiplication, e.g. ``(1e300+1j)*(nan+infj)``.  Patch by Sergey B Kirpichev.
```

Comment:
```suggestion
Correct invalid corner cases which resulted in ``(nan+nanj)`` output in complex
multiplication, e.g., ``(1e300+1j)*(nan+infj)``.  Patch by Sergey B Kirpichev.
```

Reply:
I think I always put a comma after `e.g.` even if there is a single example. We don't put a comma after `i.e.` because there cannot be more than two "things" after. We can ask @python/proofreaders for that (but I learned that after `e.g.` we always put a comma).

Reply:
Well, some statistics:
```
$ git grep 'e\.g\.[^,]' Doc/|wc -l
513
$ git grep 'e\.g\.,' Doc/|wc -l
84
```

Reply:
Oh. I'm not sure if it's good or not since standard typography tells me to put a comma... (I only learned when I was writing my thesis about this extra comma by the way).

Reply:
My understanding is also that standard English grammar has the comma usually both before and after "e.g." (and "i.e.", and "viz.") when mid-sentence.